testnets/nibirudevnet/chain.json

```diff
@@ -1,18 +1,16 @@
 {
   "$schema": "../../chain.schema.json",
-  "chain_name": "nibirudevnet",
+  "chain_name": "nibiru-devnet-1",
```

Comment:
Please do not modify any of the chain_name values. The chain_name must not be changed, and must exactly match the name of the directory it's in.